webview/src/plots/components/comparisonTable/cell/ComparisonTableCell.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react'
+import { useSelector } from 'react-redux'
+import { createSelector } from '@reduxjs/toolkit'
 import {
   ComparisonClassDetails,
+  ComparisonPlotClasses,
   ComparisonPlot
 } from 'dvc/src/plots/webview/contract'
 import { ComparisonTableLoadingCell } from './ComparisonTableLoadingCell'
 import { ComparisonTableMissingCell } from './ComparisonTableMissingCell'
 import { ComparisonTableBoundingBoxImg } from './ComparisonTableBoundingBoxImg'
 import styles from '../styles.module.scss'
 import { zoomPlot } from '../../../util/messages'
+import { PlotsState } from '../../../store'
+
+const selectPlotClasses = (state: PlotsState) => state.comparison.plotClasses
+const selectClasses = createSelector(
```

Comment:
It's good practice to add the suffix `selector` (`classesSelector`, `plotClassesSelector`)

Reply:
Done!